src/Farmer/Builders.WebApp.fs

```diff
@@ -497,10 +501,12 @@ module Converters =
             location = webApp.Location.Value
             dependsOn = webApp.Dependencies |> List.map(fun p -> p.Value)
             kind = webApp.Kind
+            https_only = webApp.HTTPSOnly
             properties =
                 {| serverFarmId = webApp.ServerFarm.Value
                    siteConfig =
                         [ "alwaysOn", box webApp.AlwaysOn
+                          "httpsOnly", box webApp.HTTPSOnly
```

Comment:
This field needs to be brought up a level, parallel with `serverFarmId` and `siteConfig` (according to [here](https://docs.microsoft.com/en-us/azure/templates/microsoft.web/sites) anyway). Also the `box` won't be necessary as the field is a boolean.